.vortex/installer/tests/Functional/Handlers/DatabaseFetchSourceHandlerProcessTest.php

```
@@ -0,0 +1,41 @@
+use DrevOps\VortexInstaller\Prompts\Handlers\AiCodeInstructions;
+use DrevOps\VortexInstaller\Prompts\Handlers\DatabaseFetchSource;
+use DrevOps\VortexInstaller\Prompts\Handlers\DatabaseImage;
+use PHPUnit\Framework\Attributes\CoversClass;
+
+#[CoversClass(DatabaseFetchSource::class)]
+#[CoversClass(DatabaseImage::class)]
+class DatabaseFetchSourceHandlerProcessTest extends AbstractHandlerProcessTestCase {
+
+  public static function dataProviderHandlerProcess(): \Iterator {
+    yield 'db_fetch_source_url' => [
+      static::cw(fn($test): string => $test->prompts[DatabaseFetchSource::id()] = DatabaseFetchSource::URL),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\DrevOps\VortexInstaller\Prompts\Handlers\DatabaseFetchSource' in method 'dataProviderHandlerProcess'. (undefined)

(StaticAccess)